### PR TITLE
Syntax themes updates.

### PR DESCRIPTION
**Pull request by the-j0k3r**
1) updated github-dark.css from 0.2.0 to [0.5.0](https://github.com/primer/github-syntax-dark/blob/master/lib/github-dark.css) please check this. also no idea what to do with `_template-github.css` please advise.

2) removed all `body.blog` from jupyter themes re: https://github.com/StylishThemes/GitHub-Dark/pull/862#issuecomment-467852136 by @Mottie

### CODE REVIEW

File: themes/src/jupyter/base16-ocean-dark.css [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
-:matches(#notebook, body.api, body.blog) .highlight .sr { color: #a3be8c !important; } /* Literal.String.Regex */
-:matches(#notebook, body.api, body.blog) .highlight .s1 { color: #a3be8c !important; } /* Literal.String.Single */
-:matches(#notebook, body.api, body.blog) .highlight .ss { color: #a3be8c !important; } /* Literal.String.Symbol */
-:matches(#notebook, body.api, body.blog) .highlight .bp { color: #eff1f5 !important; } /* Name.Builtin.Pseudo */
-:matches(#notebook, body.api, body.blog) .highlight .vc { color: #bf616a !important; } /* Name.Variable.Class */
-:matches(#notebook, body.api, body.blog) .highlight .vg { color: #bf616a !important; } /* Name.Variable.Global */
-:matches(#notebook, body.api, body.blog) .highlight .vi { color: #bf616a !important; } /* Name.Variable.Instance */
-:matches(#notebook, body.api, body.blog) .highlight .il { color: #d08770 !important; } /* Literal.Number.Integer.Long */
+#notebook .input_area, body.api pre, body.api code pre code,
+:matches(#notebook, body.api) .highlight,
```
Comment (Mottie) [inline themes/src/jupyter/base16-ocean-dark.css]: The `pre code` after `body.api code` needs to be removed in pretty much every style.

**Comment by Mottie** [comment]
And should we add a defined `.pl-corl` to every `github/` syntax theme, to fix #896?

File: themes/src/jupyter/base16-ocean-dark.css (hunk 1 above)
Comment (the-j0k3r) [reply themes/src/jupyter/base16-ocean-dark.css]: OK will do.

**Comment by the-j0k3r** [comment]
> And should we add a defined `.pl-corl` to every `github/` syntax theme, to fix #896?

That was the general ideal, but which color to which style? where do I find those values? Without it its a pita. some help would be nice.

File: themes/src/jupyter/base16-ocean-dark.css (hunk 1 above)
Comment (the-j0k3r) [reply themes/src/jupyter/base16-ocean-dark.css]: Done in https://github.com/StylishThemes/GitHub-Dark/pull/898/commits/2448cde7591d0f86dccb1f69c7463485043fb68d

**Comment by Mottie** [comment]
I'll take a look tonight or tomorrow.

**Comment by the-j0k3r** [comment]
What we doing about the `_template-<type>.css` bits, they no longer up to date at least jupyter/github side.

**Comment by Mottie** [comment]
All I did was rename one of the more robust styles to make the template.

File: themes/src/github/github-dark.css [hunk 2]
```diff
@@ -12,25 +12,25 @@
...
 .pl-smi /* variable.parameter.function, storage.modifier.package, storage.modifier.import, storage.type.java, variable.other */,
 .pl-s .pl-s1 /* string source */ {
-  color: #ddd;
+  color: #f6f8fa;
 }
 .pl-ent /* entity.name.tag */ {
   color: #7bcc72;
 }
 .pl-k /* keyword, storage, storage.type */ {
-  color: #cc2372;
```
Comment (the-j0k3r) [inline themes/src/github/github-dark.css]: test

File: themes/src/github/github-dark.css [hunk 3]
```diff
@@ -39,73 +39,74 @@
...
-  color: #ef9700;
-  background-color: #ffe3b4;
+  color: #b08800;
+  background-color: #fffdef;
 }
 .pl-mi2 /* markup.ignored, markup.untracked */ {
-  color: #d8d8d8;
-  background-color: #808080;
+  color: #2f363d;
+  background-color: #959da5;
```
Comment (the-j0k3r) [inline themes/src/github/github-dark.css]: test

```suggestion
  background-color: #959da5;
```

**Pull request merged**